source/code/source/playfab/PlayFabAndroidHttpPlugin.cpp

```diff
@@ -635,12 +628,12 @@ namespace PlayFab
             return ;
         }
 
-        jstring nameJstr = jniEnv->NewStringUTF(name);
+        auto nameJstr = jniEnv->NewStringUTF(name);
         if (nameJstr == nullptr)
         {
             return ;
         }
-        jstring valueJstr = jniEnv->NewStringUTF(value);
+        auto valueJstr = jniEnv->NewStringUTF(value);
```

Comment:
auto rules don't apply to the lines in this file. It is not obvious what the return type is for NewStringUTF.

Reply:
My previous comments weren't showing up on the review. I'm closing this one as it's an accidental duplicate.